components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -431,23 +415,6 @@ export function CreateWorkspacePage() {
                         {createWorkspaceMutation.isStarting ? "Opening Workspace ..." : "Continue"}
                     </Button>
                 </div>
-
-                {existingWorkspaces.length > 0 && !createWorkspaceMutation.isStarting && (
```

Comment:
We removed the functionallity?

Reply:
There's no context, if we still want it, right now may compare with contextUrl or `ws.status.git` as you mentioned above https://github.com/gitpod-io/gitpod/pull/19022#discussion_r1390888313

Reply:
yeah, we may not break dashboard, only migrate